scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html

```diff
@@ -206,8 +206,8 @@ <h3 class="fs-subtitle">The mode of inheritance specific to the variant-disease
               </fieldset>
 
               <fieldset data-step="5">
-                <legend>Clinical Significance</legend>
-                <h3 class="fs-subtitle">The interpretation, or clinical significance, of the variant for the submitted condition</h3>
+                <legend>Germline Classification</legend>
+                <h3 class="fs-subtitle">Represents a variant-level classification for a disease, rather than an interpretation of the clinical significance of a variant for a specific patient.</h3>
```

Comment:
Taken from what's written on their repo: https://github.com/ncbi/clinvar/blob/master/submission_templates/README.md

Reply:
Great. Something tells me there will be more changes before this is quite finalised. 😂